modules/standard/Math.chpl

```diff
@@ -1290,6 +1290,20 @@ module Math {
     extern proc yn(n: c_int, x: real(64)): real(64);
     return yn(n.safeCast(c_int), x);
   }
+  
+  /* Returns 1 if the sign of `x` is negative, else returns 0. */
```

Comment:
The chpldoc should say something about why this is useful e.g. mention infinity, nan, and/or signed zero